app/controllers/users_controller.rb

```diff
@@ -142,8 +143,29 @@ def access_list_approvals
   def email
   end
 
+  # GET /facilities/:facility_id/users/:id/edit
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  # PUT /facilities/:facility_id/users/:id
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(edit_user_params)
```

Comment:
CodeClimate is flagging this as unprotected mass assignment. It looks like we're using `require`/`permit` as expected. I think it's because we don't have `include ActiveModel::ForbiddenAttributesProtection` for the user model. 

I'd like to get that included as part of this story, but I'm not sure how broad an impact it would have (it would likely affect the other actions in this controller, but I don't know where else it might have effects).

Do you think you could do a quick estimate of how much work you think this would take? Then we can decide if it's worth pursuing. If we decide to do it, I would be comfortable doing that in a separate PR.